lib/generators/templates/spec.erb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+RSpec.describe <%= class_name %> do
+  describe 'call' do
```

Comment:
The best practice is to only have a single public method call on interactors I think it's much more beneficial to provide a basic template for testing that method. These specs are still pending.